sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileIO.java

```diff
@@ -373,6 +373,7 @@ public static Match match() {
   public static MatchAll matchAll() {
     return new AutoValue_FileIO_MatchAll.Builder()
         .setConfiguration(MatchConfiguration.create(EmptyMatchTreatment.ALLOW_IF_WILDCARD))
+        .setOutputParallelization(false)
```

Comment:
If OutputParallelization means adding a reshuffle, then we default to true here to keep default behavior the same